browser-only.js

```diff
@@ -39,6 +39,9 @@ if (typeof window !== 'undefined') {
   require('shim-keyboard-event-key');
 }
 
+// :focus-visible shim
+require('focus-visible');
```

Comment:
Looks like it does its own `window` and `document` checks so I didn't need to include it in the block above.

https://github.com/WICG/focus-visible/blob/4299b3e7315491fba99f1efb1b42b3fafafd6fd4/src/focus-visible.js#L278

https://github.com/WICG/focus-visible/blob/4299b3e7315491fba99f1efb1b42b3fafafd6fd4/src/focus-visible.js#L299